app/views/ht_users/show.html.erb

```diff
@@ -5,6 +5,8 @@
   <% expired_text = @user.expired? ? "Expired" : "Expires"
      expires_class = @user.expiring_soon? ? "expiring-soon" : ""
   %>
+  <% req = HTApprovalRequest.where(userid: @user.email).first %>
```

Comment:
Same issue as in the index view above - extract this to a user presenter?

Reply:
Looks like HTUserPresenter is working for this purpose.